stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -37,6 +38,7 @@
 import java.util.List;
 
 final class AndroidDOMProvider implements DOMProvider, AndroidDescriptorHost {
+  private static final long REPORT_CHANGED_INTERVAL_MS = 1000;
```

Comment:
I'm not really sure why we'd need a javadoc for this. I do like the idea of moving this constant (and the comment) over so it's grouped with the associated boolean and the Runnable.